sdk/attestation/Azure.Security.Attestation/tests/AttestationClientTestEnvironment.cs

```diff
@@ -19,16 +19,26 @@ public class AttestationClientTestEnvironment : TestEnvironment
 
         public string SharedAttestationUrl => "https://shared" + LocationShortName + "." + LocationShortName + ".test.attest.azure.net";
 
+#if NET9_0_OR_GREATER
```

Comment:
[SYSLIB0057: X509Certificate2 and X509Certificate constructors for binary and file content are obsolete](https://learn.microsoft.com/en-us/dotnet/fundamentals/syslib-diagnostics/syslib0057)